firmware/spiders/asus.py

```diff
@@ -1,3 +1,4 @@
+import re
```

Comment:
```suggestion
from re import findall
```
You can not know that, but we prefer pulling in the methods we use into the file's root namespace. If that is okay, please consider accepting my suggestion here :-)